Automattic/wp-calypso
# Site Selector: Add Backup Plugin sites to Cloud site selector

#### Changes proposed in this Pull Request

* Adds Backup Plugin sites without Jetpack plugin to the Cloud site selector

#### Testing instructions



* Add the Backup Plugin to a test site without the Jetpack plugin
* Ensure it appears in a cloud instance running this branch in the site selector
* http://jetpack.cloud.localhost:3000/landing
